app/src/main/java/org/piramalswasthya/sakhi/adapters/FormInputAdapterOld.kt

```
@@ -245,6 +251,47 @@ class FormInputAdapterOld(
+        ViewHolder(binding.root) {
+        companion object {
+            fun from(parent: ViewGroup): ViewHolder {
+                val layoutInflater = LayoutInflater.from(parent.context)
+                val binding = RvItemFormUploadImageBinding.inflate(layoutInflater, parent, false)
+                return FileUploadInputViewHolder(binding)
+            }
+        }
+
+        fun bind(item: FormInputOld, isEnabled: Boolean) {
+
+        }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)